.eslintrc

```
@@ -0,0 +1,109 @@
+{
+	"env": {
+		"browser": true,
+		"commonjs": true,
+	},
+	"extends": "eslint:recommended",
+	"globals": {
+		"Prism": "readonly",
+	},
+	"parserOptions": {
+		"ecmaVersion": 5,
+	},
+	"root": true,
+	"rules": {
+		"comma-dangle": [
+			"warn",
+			"only-multiline",
+		],
+		"linebreak-style": [
+			"error",
+			"unix",
+		],
+		"no-console": "off",
+		"no-empty": "off",
```

An empty block is usually a sign that you forgot to do something, so I think that this should be a warning.

One notable exception is [empty `catch` blocks](https://eslint.org/docs/rules/no-empty#allowemptycatch).

```suggestion
		"no-empty": [
			"warn",
			{
				"allowEmptyCatch": true
			}
		],
```